keras_retinanet/models/vgg.py

```diff
@@ -0,0 +1,72 @@
+"""
+Copyright 2017-2018 cgratie (https://github.com/cgratie/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+import keras
+
+from ..models import retinanet
+
+
+custom_objects = retinanet.custom_objects.copy()
+
+
+def download_imagenet(backbone):
+    backbone = int(backbone.replace('vgg', ''))
+
+    if backbone == 16:
+        resource = keras.applications.vgg16.WEIGHTS_PATH_NO_TOP
+        checksum = '6d6bbae143d832006294945121d1f1fc'
+    elif backbone == 19:
+        resource = keras.applications.vgg19.WEIGHTS_PATH_NO_TOP
+        checksum = '253f8cb515780f3b799900260a226db6'
+    else:
+        resource = None
+        checksum = None
+
+    weights_path = keras.applications.imagenet_utils.get_file(
+        'vgg{}_weights_tf_dim_ordering_tf_kernels_notop.h5'.format(backbone),
```

Comment:
This would change a little bit.

Reply:
Not too much, the `vgg` beginning would be read from the backbone, so I just take it out from the format string.